Say Their Names/Source/View/Home/Person/PersonCell.swift

```diff
@@ -8,11 +8,28 @@
 
 import UIKit
 
-class PersonCell: UICollectionViewCell {
+final class PersonCell: UICollectionViewCell {
 
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var imgView: UIImageView!
+    @IBOutlet weak var dobLabel: UILabel!
+    @IBOutlet weak var bookmarkButton: UIButton!
```

Comment:
Always make them private.
Avoid raw variables, e.g: `imgView`, `dobLabel`, always use the fullname. `imageView` or `birthdayLabel`. In this case, I guess the `imageView`, will be the user's profile picture, then, it might be `profileImageView`.

```suggestion
    @IBOutlet weak private var nameLabel: UILabel!
    @IBOutlet weak private var profileImageView: UIImageView!
    @IBOutlet weak private var birthdayLabel: UILabel!
    @IBOutlet weak private var bookmarkButton: UIButton!
```